varya/assets/sass/child-theme/variables.css

```diff
@@ -67,13 +67,6 @@
 	--global--spacing-measure: unset;
 	--global--spacing-horizontal: 25px;
 	--global--spacing-vertical: 30px;
-	/* Elevation */
-	--global--elevation-1: 0px 0px 4px 2px rgba( 0, 0, 0, 0.2 );
```

Comment:
The nav block will also have a hover state, and it would be nice to leverage the same elevation variable. Unless we don't plan on adding any elevation there, but that seems inconsistent.

Reply:
Hmm, you're right. Maybe it makes sense to have just a single box-shadow rule?

Reply:
In any case, maybe we should save this for when we style that in a PR.

Reply:
Yeah I think saving one `--global--elevation` variable would be useful!